colanderalchemy/schema.py

```diff
@@ -130,14 +148,17 @@ class MyModel(Base):
 
         # Obtain configuration specific from the mapped class
         kwargs.update(getattr(self.inspector.class_, self.ca_class_key, {}))
+        includes = includes or kwargs.pop('includes', {})
+        excludes = excludes or kwargs.pop('excludes', {})
+        overrides = overrides or kwargs.pop('overrides', {})
```

Comment:
Is there any reason this can't be:
```
self.includes = includes or kwargs.pop('includes', {})
self.excludes = excludes or kwargs.pop('excludes', {})
self.overrides = overrides or kwargs.pop('overrides', {})
```
And then the lower assignment removed?